canonical/snapcraft.io
# Build model view

## Done
Built the model detail view

## How to QA
- Go to https://snapcraft-io-4344.demos.haus/login-beta and login
- Go to https://snapcraft-io-4344.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models
- Click through to any model
- Check that the data is there
- Check that you can change the API key

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-4737
